### PR TITLE
Text hidden

### LINKED ISSUE #684: Some text are hidden in "A little more about FOSSASIA"
Screenshot (182)

### PR DESCRIPTION
**Pull request by Sohan022**
Some text are hidden in "A little more about FOSSASIA" corrected.
fixed: #684 
preview link: https://sohan022.github.io/fossasia.org/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1116,7 +1116,7 @@ <h1>With the support of many people and companies around the world we have achie
 						</li>
 
 						<li>
-							<a href="img/DSC_0502_1_0.JPG" data-lightbox="true" data-title="Lightbox Image">
```
Comment (Chaitanyassr) [inline index.html]: Why changing the name of the image?

**Review by Chaitanyassr — CHANGES_REQUESTED** [review]
Greate, Please squash the commit.

**Comment by Sohan022** [comment]
I'm creating a new pull request due to some mistake in this branch of the repo.

**Comment by Chaitanyassr** [comment]
> I'm creating a new pull request due to some mistake in this branch of the repo.

Close the old one.

**Review by Raunaksingh100 — COMMENTED** [review]
You can closed this pr because you create a new pr for this same issue

**Pull request closed without merging**